sky/serve/load_balancer.py

```diff
@@ -160,11 +160,28 @@ async def _proxy_with_retries(
         # SkyServe supports serving on Spot Instances. To avoid preemptions
         # during request handling, we add a retry here.
         retry_cnt = 0
+        # Here we try to not retry those failed replicas for the case when the
+        # replica is in a NOT_READY state but does not sync-ed to the load
+        # balancer yet. However, we still maintain a per-request failed replica
+        # list instead of the global one to avoid the case for transient
+        # networking issues, and letting new requests to retry them. Since our
+        # LB policy is a global one, when the request rate is high, it is likely
+        # that multiple retries on a single request will use the same replica.
+        # Here we use the failed replica list to keep track of the failures
+        # happened on every request and try to avoid them in the next retry.
```

Comment:
This looks awesome! Thanks

Reply:
I change
`to allow multiple requests to still try xxx for one request`
to
`to allow multiple requests to still try xxx for one time`
since the former is slightly confusing (multiple req for one req).